h/services/annotation_delete.py

```diff
@@ -21,6 +22,12 @@ def delete(self, annotation):
         """
         annotation.updated = datetime.utcnow()
         annotation.deleted = True
+        self.job_queue.add_by_id(
+            name="sync_annotation",
+            annotation_id=annotation.id,
+            tag="AnnotationDeleteService.delete_annotation",
+            schedule_in=60,
+        )
```

Comment:
Add a `"sync_annotation"` job to the queue when deleting an annotation, as we do when creating or updating one.

This is for when deleting a single annotation via the API. In a future PR this is how we'll also handle bulk deletions of annotations from Elasticsearch when deleting a user: we'll add a job for each of the user's annotations (in batches: we won't try to add all the jobs at once).